candump.c

```diff
@@ -69,13 +69,14 @@
 
 /* for hardware timestamps - since Linux 2.6.30 */
 #ifndef SO_TIMESTAMPING
-#define SO_TIMESTAMPING 37
+#define SO_TIMESTAMPING 50 
```

Comment:
??
This is either wrong and doesn't fit to this patch.
https://elixir.bootlin.com/linux/v5.9.13/source/include/uapi/asm-generic/socket.h#L127

Reply:
Sorry that is a typo.  I'm going to need some clang-tidy and rebase workflow optimizations in my near future to stop missing easy stuff.